src/main/java/com/jcabi/jdbc/JdbcSession.java

```diff
@@ -309,6 +310,25 @@ public void commit() throws SQLException {
     }
 
     /**
+     * Call an SQL stored procedure.
+     *
+     * <p>JDBC connection is opened and, optionally, closed by this method.
```

Comment:
@mkordas the ``run`` method does that in the ``finally`` block. All the other methods, e.g. ``execute()`` say that in the javadoc, so I left it there.